packages/rule-no-bom/src/no-bom.ts

```diff
@@ -47,7 +47,7 @@ export default class implements IRule {
             const content = request.response.body.rawContent;
 
             if (content[0] === 0xEF && content[1] === 0xBB && content[2] === 0xBF) {
-                await context.report(resource, element, `Text based files shouldn't start with the BOM character to force UTF-8 encoding`);
+                await context.report(resource, element, `Text based resources shouldn't start with the BOM character to force UTF-8 encoding`);
```

Comment:
> resource 

The error resource to just 1 not many.

Reply:
Also this commit should start with `Breaking:` since we are changing the error message.